lte/gateway/c/oai/lib/hashtable/hashtable.c

```diff
@@ -490,7 +490,7 @@ hashtable_element_array_t* hashtable_ts_get_elements(
     return NULL;
   }
   ea           = calloc(1, sizeof(hashtable_element_array_t));
-  ea->elements = calloc(hashtblP->num_elements, sizeof(hash_key_t*));
+  ea->elements = calloc(hashtblP->num_elements, sizeof(void*));
 
   while ((ea->num_elements < hashtblP->num_elements) && (i < hashtblP->size)) {
```

Comment:
use of uninitialized "ea->num_elements". same for "ka->num_keys" above.
Fine if you want to resolve these in a different PR

Reply:
Did you resolve this in a follow up PR yourself, or is it open / needs eyes on it? Thanks!

Reply:
I did not. Please feel free to submit a PR